app/assets/javascripts/code/values/session.js

```diff
@@ -9,3 +9,27 @@ export const formatSessionForList = session => ({
   timeframe: session.timeframe,
   shortTypes: session.shortTypes
 });
+
+export class Session {
```

Comment:
Personally I'd avoid using `class`.  What I'd suggest is something like
```
export const roundedAverage = session => Math.round(average(session))

export const average = session => session.average

...
```
Then you can

```
import Session from '/path'

Session.average(session)
```

Reply:
There's a lot of ceremony with `class`. Also it introduces `this`. And it's less composable than functions

Reply:
I tried something like this, but I was getting this error:
`TypeError: Cannot read property 'roundedAverage' of undefined`
 `at MobileSessions.drawSessionInLocation `
In the end I was able to make it work with class